site/sfguides/src/intra_company_data_sharing_with_the_snowflake_internal_marketplace/intra_company_data_sharing_with_the_snowflake_internal_marketplace.md

```diff
@@ -0,0 +1,1070 @@
+
+This Quickstart is focused on the <mark>Access</mark> pillar.
+
+The objective of the Access pillar is to make it simple to share, discover, understand/build trust and access listings across any boundary, internal or external to the organization, and to make loose objects discoverable across account boundaries within an organization, supported by the tools necessary to ensure policy compliance, security, and data quality.
+
+In this lab you will experience the latest **Snowflake Horizon Access pillar** features for sharing data and native apps intra-company: organizational listings, unified search & discovery, data quality monitoring, role-based governance policies and programmatic management of data products. We will cover structured and unstructured data that is stored on-platform or on external storage.
+
+### What You’ll Learn
+
+- How to blend TastyBytes Point-of-Sale and Marketplace Weather data to build analytics data products, then publish Listings targeted at accounts in your company
+- How to configure data privacy polices that are preserved in the consumer accounts that install the listings. Tag-based column masking, row-access, aggregation and projection policies will be created with database roles.
+- How to setup Data Metrics Functions to monitor data quality of the shared data products
+
+### What You’ll Need
+
+- Basic knowledge of SQL, Database Concepts, Snowflake [Listings](https://other-docs.snowflake.com/en/collaboration/collaboration-listings-about)
+- Familiarity with [Snowsight Worksheets](https://docs.snowflake.com/en/user-guide/ui-snowsight-worksheets-gs)
+
+### What You’ll Build
+
+- Analytics data products for TastyBytes, that models a global food truck network with localized menu options in 15 countries, 30 major cities and 15 core brands
+- Listings comprised of metadata, data and application code, targeted at accounts in the same organization but in different cloud regions
+- Governance policies based on shared role-based-access-controls that are enforced at the target account consuming the listing
+- Install listings containing Iceberg Tables, then blend with local datasets to derive insights
+<!--
+- Exploration of all shared and local data with Universal Search and Copilot
+-->
+
+### Prerequisites
+
+#### Create 3 Snowflake Trial Accounts in the same Snowflake Organization
+
+Signup for an AWS trial account [here](https://signup.snowflake.com/)
+
+- Choose **AWS** as cloud provider, **Business Critical** edition, **AWS_US_WEST_2 (Oregon)** region
+- Activate trial account with admin user <mark>horizonadmin</mark>
+  - admin user has system roles: ACCOUNTADMIN, ORGADMIN, SYSADMIN
+- Login and create a SQL Worksheet named _**Account Setup**_
+
+<ins>Note:</ins> alternatively you _can_ use an existing account instead of a trial account, provided that account has the `ORGADMIN` system role enabled.
+
+- Create user `horizonadmin`, grant it ACCOUNTADMIN and ORGADMIN roles.
```

Comment:
@sfc-gh-hnielsen The SQL statements to execute should be separated out into a new step labeled "Setup".